hisat2.2.1/1.0.0/unittest.yml

```diff
@@ -2,4 +2,34 @@ commands:
   - cmd: "hisat2 --version | head -n 2"
     expect_text: |
       /usr/bin/hisat2-2.2.1/hisat2-align-s version 2.2.1
-      64-bit
\ No newline at end of file
+      64-bit
+  - cmd: "gtfToGenePred 2>&1 | sed -e '/^$/d; s/^[ \t]*//; s/[ \t]*$//'"
```

Comment:
Why don't you do a `| head -n 1` here so you don't have to check the entire output?

Reply:
To verify that it also installed the 64-bit version.  The rest of the output also would be harder to write, due to the number of special characters.

Reply:
@GHAStVHenry finished this change, and the ones below, and the tests passed.